src/blog/2023/005-form-js-1.x.md

```diff
@@ -0,0 +1,111 @@
+</p>
+
+<!-- continue -->
+
+We dedicate the theme to helping you build complex and reusable forms with ease. Try out the new features [on our demo](https://demo.bpmn.io/form) or using the [Camunda form playground](https://github.com/camunda/form-playground).
+
+## New layouting capabilities
+
+With the addition of group components, you can now organize form components seamlessly. You can add and arrange other form components into the group, like in the parent form.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/005-group.png">
+
+  <p class="caption">
+    Create groups to organize form components logically.
+  </p>
+</div>
+
+Resizable form fields allow you to adjust the size of form components, creating visually appealing form layouts.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/005-resize-field.gif">
+
+  <p class="caption">
+    Resize form fields to fit the layout.
+  </p>
+</div>
+
+Additionally, introducing the spacer component enables you to create white space between form components, making it possible to place them more freely.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/005-spacer.png">
+
+  <p class="caption">
+    Use spacers to align form components to the right.
+  </p>
+</div>
+
+## New data binding capabilities
+
+Data binding receives a significant upgrade, expanding the possibilities of providing nested form keys.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/005-nested-key.png">
+
+  <p class="caption">
+    Use nested form keys to bind structured data.
+  </p>
+</div>
+
+Binding groups to a data path enables the provision of additional context and enhances the efficiency of organizing form data.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/005-group-path.png">
+
+  <p class="caption">
+    Specify a data path to a group to provide more context.
+  </p>
+</div>
+
+Without manual work, dynamic lists enable the effortless generation of repetitive sets of form components, such as data tables or ordered lists.
+
+<div class="figure full-size">
+  <img src="{{ assets }}/attachments/blog/2023/005-dynamic-list.png">
+
+  <p class="caption">
+    Dynamic lists allow to model repetitive lists of form components.
+  </p>
+</div>
+
+## Controllable and reusable forms via expression properties
+
+By leveraging FEEL - the expression language [standardized by the OMG](http://www.omg.org/spec/DMN/) - and [FEEL templating](https://github.com/bpmn-io/feelers), you can streamline the development process and ensure consistency across a wide range of forms, making complex form creation more efficient and reusable.
+
+These releases let you dynamically control various form properties, including read-only settings, labels, descriptions, multi-select options, and minimum and maximum validation.
```

Comment:
Maybe something like "As part of these releases and moving forward, we are allowing you to define more and more properties via FEEL...", or something along those lines.